app/services/encrypted_document_storage/document_writer.rb

```diff
@@ -0,0 +1,35 @@
+module EncryptedDocumentStorage
+  class DocumentWriter
+    def encrypt_and_write_document(front_image:, back_image:)
+      key = SecureRandom.bytes(32)
+      encrypted_front_image = aes_cipher.encrypt(front_image, key)
+      encrypted_back_image = aes_cipher.encrypt(back_image, key)
+
+      front_image_reference = SecureRandom.uuid
```

Comment:
how would we feel about calling this `front_image_id` instead of reference?

Reply:
or `front_image_uuid`

Reply:
Shuffled things around and did some renaming